docs/content/dagster-cloud/developing-testing/managing-deployments.mdx

```diff
@@ -0,0 +1,193 @@
+
+<Image
+alt="The deployment switcher in Dagster Cloud"
+src="/images/dagster-cloud/developing-testing/deployments/deployment-switcher.png"
+width={660}
+height={207}
+/>
+
+To manage deployments, click **View all deployments**. **Note**: Only users with [**Organization Admin** permissions](/dagster-cloud/account/managing-users#understanding-user-permissions) will see this option in the drop-down.
+
+---
+
+## Creating deployments
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users#understanding-user-permissions">
+    Organization Admin permissions
+  </a>{" "}
+  are required to create deployments. Additionally, note that creating multiple
+  deployments requires an{" "}
+  <a href="https://dagster.io/pricing" target="new">
+    Enterprise plan
+  </a>
+  .
+</Note>
+
+To create a deployment:
+
+1. Sign in to your Dagster Cloud account.
+2. Access the **Deployments** page using on eof the following options:
+   - Click the **deployment switcher > View all deployments**.
+   - Click **your user icon > Cloud Settings**.
+3. Click the **+ New deployment** button.
+4. In the modal that displays, fill in the following:
+   - **Name** - Enter a name for the deployment.
+   - **Initial deployment permissions** - Select the permissions you want to use to create the deployment:
+     - **Empty permissions** - Creates the deployment with an empty set of permissions. **Note**: Only Organization Admins will be able to manage the deployment until other uses are granted Admin or Editor permissions.
+     - **Copy from** - Creates the deployment using permissions duplicated from an existing deployment.
+5. When finished, click **Create deployment**.
+
+---
+
+## Deleting deployments
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users#understanding-user-permissions">
+    Organization Admin permissions
+  </a>{" "}
+  are required to delete deployments. Additionally, note that deleting a
+  deployment also deletes all its associated data, including code locations,
+  jobs, schedules, and sensors.
+</Note>
+
+To delete a deployment:
+
+1. Sign in to your Dagster Cloud account.
+2. Access the **Deployments** page using on eof the following options:
+   - Click the **deployment switcher > View all deployments**.
+   - Click **your user icon > Cloud Settings**.
+3. Click the **Delete** button next to the deployment you want to delete.
+4. When prompted, confirm the deletion.
+
+---
+
+## Configuring deployment settings
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users#understanding-user-permissions">
+    Organization Admin permissions
+  </a>{" "}
+  are required to modify deployment settings.
+</Note>
+
+Deployment settings can be configured in the Dagster Cloud interface or using the dagster-cloud CLI. Refer to the [deployment settings reference](/dagster-cloud/developing-testing/deployment-settings-reference) for more info about individual settings.
```

Comment:
should we make it clear that these settings only apply to full deployments? (even though this is in the Full Deployments section)

Reply:
I'm inclined to say it's not necessary, given that we specifically state this guide is for full deployments before this.